akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -412,7 +417,63 @@ TODO
 Disabling the Java Serializer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-TODO
+With Artery it is possible to completely disable Java Serialization for the entire Actor system.
+
+Java serialization is known to be slow and prone to attacks of various kinds - it never was designed for high 
+throughput messaging after all. However it is very convenient to use, thus it remained the default serialization 
+mechanism that Akka used to serialize user messages as well as some of its internal messages in previous versions.
+Since the release of Artery, Akka internals do not rely on Java serialization anymore (one exception being ``java.lang.Throwable``).
```

Comment:
So I'd mention throwable anyway, so people are aware to replace serialization for it if disabling java s. entirely. Adding note on remote deployment